src/components/ListComposable/utils/flattenItems.ts

```diff
@@ -0,0 +1,40 @@
+import type {ListItemId, ListItemType} from '../types';
+
+import {getGroupedId} from './groupedIdUtils';
+
+export function flattenItems<T>(
+    items: ListItemType<T>[],
+    groupsExpandedState: Record<ListItemId, boolean> = {},
+): ListItemId[] {
+    console.time('flattenItems');
```

Comment:
U can use this [approach](https://github.com/gravity-ui/uikit/blob/b9bda434a8a3722c088758efbf990f63cd221e62/src/components/utils/FocusTrap.tsx#L128) to hide logging in production